src/arch_ppc.py

```diff
@@ -409,13 +415,51 @@ def normalize_instruction(cls, instr: AsmInstruction) -> AsmInstruction:
     def parse(
         cls, mnemonic: str, args: List[Argument], meta: InstructionMeta
     ) -> Instruction:
+        inputs: List[Access] = []
+        outputs: List[Access] = []
+        clobbers: List[Access] = []
         jump_target: Optional[Union[JumpTarget, Register]] = None
         function_target: Optional[Union[AsmGlobalSymbol, Register]] = None
         is_conditional = False
         is_return = False
 
+        cr0_bits: List[Access] = [
+            Register("cr0_lt"),
+            Register("cr0_gt"),
+            Register("cr0_eq"),
+            Register("cr0_so"),
+        ]
+        memory_sizes = {
+            "b": 1,
+            "h": 2,
+            "w": 4,
+            "mw": 4,
+            "fs": 4,
+            "fd": 8,
+        }
+        size = memory_sizes.get(mnemonic.lstrip("stl").rstrip("azux"))
+
+        def make_memory_access(arg: Argument) -> Access:
+            assert size is not None
+            if isinstance(arg, AsmAddressMode):
+                return MemoryAccess(
+                    base_reg=arg.rhs,
+                    offset=arg.lhs,
+                    size=size,
+                )
+            elif isinstance(arg, AsmGlobalSymbol):
```

Comment:
also need to support BinOp (symbol + offset). Should probably mirror mips's version of make_memory_access and run this if unconditionally